deploy/deploy_citus_server.yml

```diff
@@ -0,0 +1,58 @@
+# deploy_citus_server.yml
+---
+- hosts: citus
+  become: yes
+  become_method: sudo
+  roles:
+    - {role: iptables, when: "iptables_config"}
+    - {role: common, when: "install_glusterfs"}
+    - {role: epel, when: "use_epel and not 'production' in group_names"}
+    - {role: log-courier, when: "install_logstash"}
+    - role: jdk
+    - role: postgresql-server
+  tasks:
+    - name: Ensure file transfer folder exists
+      file:
+        dest: "{{ pp_file_transfer_dir }}"
+        state: directory
+        mode: 0755
+        owner: postgres
+        group: postgres
+
+    - name: Create symlink from PPAS data directory
+      file:
+        dest: "{{ pp_datadir }}/file_transfers"
+        src: "{{ pp_file_transfer_dir }}"
+        state: link
+
+    - name: Setup host file
+      lineinfile:
+        dest: /etc/hosts
+        regexp: '[[0-9+]\.]+\s+{{ item }}'
+        line: "{{ hostvars[item]['ansible_ssh_host'] }} {{ item }}"
+      with_items: "{{ groups['citus'] }}"
+      when: not custom_repo
+
+
+- hosts: citus
+  roles:
+    - role: postgresql-server-config
+    - role: citus
```

Comment:
I had to do some rearranging here.  Basically, since the "citus" library is added as part of the postgresql.conf template now, Postgres won't start unless Citus is already installed.  So, I had to remove the task for starting the postgresql server from the postgresql-server role, and put it in the playbook instead (which would have to be done in all playbooks using the role).  Then, I had to move the citus role up to run right after postgresql-server, and then postgresql-server-config after that, all in one `roles` list.  I then merged the `tasks` and `post_tasks` into a single `post_tasks` block.  This ensures that the handler isn't run until after Citus has been installed, since that would fail as well.  Let me know if this is unclear!